### PR TITLE
[15.0][MIG] product_pricelist_direct_print_website_sale: Migration to version 15.0

### PR DESCRIPTION
**Pull request by pilarvargas-tecnativa**
TT39001 @Tecnativa

@chienandalu @CarlosRoca13 please review

### CODE REVIEW

File: product_pricelist_direct_print_website_sale/report/product_pricelist_xlsx.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+    def _add_extra_header(self, sheet, book, next_col, header_format):
+        next_col = super()._add_extra_header(sheet, book, next_col, header_format)
+        if book.show_public_category:
+            next_col += 1
+            sheet.write(5, next_col, _("Public Category"), header_format)
+        return next_col
+
+    def _add_extra_info(self, sheet, book, product, row, next_col, **kw):
+        next_col = super()._add_extra_info(sheet, book, product, row, next_col)
```
Comment (CarlosRoca13) [inline product_pricelist_direct_print_website_sale/report/product_pricelist_xlsx.py]:
```suggestion
        next_col = super()._add_extra_info(sheet, book, product, row, next_col, **kw)
```
Please keep the `**kw` argument when calling the super method

**Review by CarlosRoca13 — CHANGES_REQUESTED** [review]
Please squash the administrative commits

File: product_pricelist_direct_print_website_sale/report/product_pricelist_xlsx.py (hunk 1 above)
Comment (pilarvargas-tecnativa) [reply product_pricelist_direct_print_website_sale/report/product_pricelist_xlsx.py]: Administrative commits squashed and the argument is keeping :)

**Review by chienandalu — APPROVED** [review]
LGTM Thanks! :+1:

**Review by CarlosRoca13 — APPROVED** [review]

**Comment by pedrobaeza** [comment]
/ocabot migration product_pricelist_direct_print_website_sale
/ocabot merge nobump

**Pull request merged**